docs/mod_tap.md

```diff
@@ -61,6 +61,78 @@ You may also run into issues when using Remote Desktop Connection on Windows. Be
 To fix this, open Remote Desktop Connection, click on "Show Options", open the the "Local Resources" tab, and in the keyboard section, change the drop down to "On this Computer". This will fix the issue, and allow the characters to work correctly.
 It can also be mitigated by increasing [`TAP_CODE_DELAY`](config_options.md#behaviors-that-can-be-configured).
 
+## Intercepting Mod-Taps
+
+### Changing tap function
+
+Basic keycode limitation with Mod-Tap can be worked around by intercepting it in `process_record_user`. For example, shifted keycode `KC_DQUO` cannot be used with `MT()` because it is a 16-bit keycode alias of `LSFT(KC_QUOT)`. But the following custom code can be used to intercept the "tap" function to manually send `KC_DQUO`:
+
+```c
+bool process_record_user(uint16_t keycode, keyrecord_t *record) {
+    switch (keycode) {
+        case LCTL_T(KC_DQUO):
+            if (record->tap.count && record->event.pressed) {
+                tap_code16(KC_DQUO); // Send KC_DQUO on tap
+                return false;        // Return false to ignore further processing of key
+            }
+            break;
+    }
+    return true;
+}
+```
+
+### Changing hold function
+
+Likewise, the same custom code can also be used to intercept the hold function to send custom user key code. The following example uses `LT(0, kc)`, a current-layer Mod-Tap with no practical use, to add cut, copy and paste function to X,C and V keys when they are held down:
```

Comment:
`LT()` is not a “current-layer Mod-Tap” (but it is really useless by itself, because layer 0 is always active, therefore the hold action of `LT(0, kc)` would actually have no effect).

And maybe there should be a warning against using `MT()` codes for this purpose, unless `IGNORE_MOD_TAP_INTERRUPT` is defined (intercepting the tap function is not as problematic — it just effectively enables `IGNORE_MOD_TAP_INTERRUPT` for that key, but it the tap function is left unchanged, and `IGNORE_MOD_TAP_INTERRUPT` is not defined, the default tap handling code may try to perform the standard hold action in some cases).